build/dev-server.js

```diff
@@ -22,7 +22,7 @@ var autoOpenBrowser = !!config.dev.autoOpenBrowser
 // https://github.com/chimurai/http-proxy-middleware
 var proxyTable = config.dev.proxyTable
 
-var app = express()
+var app = require('../server/index.js')
 var compiler = webpack(webpackConfig)
 
 var devMiddleware = require('webpack-dev-middleware')(compiler, {
```

Comment:
That's not expected to be used.

Reply:
So server and client should be completely seperate?

Reply:
So server and client should be completely seperate?

Reply:
That would probably be the best idea, but I haven't had time to check SSR on Vue.